src/spot-client.ts

```diff
@@ -1,11 +1,10 @@
 import { AxiosRequestConfig } from 'axios';
-import { KlineInterval } from './types/shared';
+import { KlineInterval, APIResponse, SymbolInfo } from './types/shared';
 import { NewSpotOrder, OrderSide, OrderTypeSpot, SpotOrderQueryById } from './types/spot';
-import BaseRestClient from './util/BaseRestClient';
 import { GenericAPIResponse, getRestBaseUrl, RestClientOptions } from './util/requestUtils';
 import RequestWrapper from './util/requestWrapper';
 
-export class SpotClient extends BaseRestClient {
+export class SpotClient {
```

Comment:
```suggestion
export class SpotClient extends BaseRestClient {
```

I want to keep extending base rest client, so it can enforce implementation of getServerTime() resolving to a number, as well as being able to make requests with this.get/post/delete/etc()